sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -117,15 +117,14 @@ def get_responses():
   def _execute(self, task, request):
     try:
       response = task()
-    except Exception as e:  # pylint: disable=broad-except
-      traceback.print_exc(file=sys.stderr)
+    except Exception:  # pylint: disable=broad-except
+      traceback_string = traceback.format_exc()
+      print(traceback_string, file=sys.stderr)
```

Comment:
This should print an exactly equivalent message to what we had before. I just didn't see any point in formatting the traceback multiple times.